guide/designing-products/user-research.md

```diff
@@ -42,9 +42,9 @@ Illustration sources
 
 # User research
 
-We design digital products for the benefit of others. To do that, we need to understand who those others are and their unique needs, desires, and behaviors. A comprehensive design strategy balances user needs with considerations around the technology, business models, societal and economic impact, and other factors. In this section, we focus on understanding the users better.
+We design products to benefit users. That’s why it is necessary to understand their needs, desires, and behaviors.
```

Comment:
I think this removes the important point that design is not solely about users. It is often about balancing user needs with those other aspects (tech, business model, etc). Bitcoin is a prime example, as the tech is highly opinionated and forces users to change certain behaviors.

Reply:
Fixed in https://github.com/BitcoinDesign/Guide/pull/365/commits/7a005dfb2b53e00b399f33ff33963a3f2a505a3d